test/headers/mock_usart.hpp

```diff
@@ -0,0 +1,54 @@
+#pragma once
```

Comment:
i believe there is a mock usart in the library at the moment

Reply:
There is, but that implementation doesn't have the needed behaviour.
This implementation had the receive() method just always return 0xAA.

https://github.com/R2D2-2019/USART_library/issues/10

Reply:
this will be changed shortly we will pick this up with the new team